packages/ckeditor5-ui/theme/components/form/form.css

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+@import "@ckeditor/ckeditor5-ui/theme/mixins/_rwd.css";
+@import "@ckeditor/ckeditor5-ui/theme/mixins/_dir.css";
+
+:root {
+	--ck-form-default-width: 340px;
+}
+
+.ck.ck-form {
+	padding: 0 0 var(--ck-spacing-large);
+
+	&.ck-form_default-width {
+		width: var(--ck-form-default-width);
+	}
+
+	&:focus {
+		outline: none;
+	}
+
+	& .ck.ck-form__row {
+		&.ck-form__row_large-top-padding {
+			padding-top: var(--ck-spacing-large);
+		}
+
+		&.ck-form__row_large-bottom-padding {
+			padding-bottom: var(--ck-spacing-large);
+		}
+
+		&.ck-form__row_with-submit {
+			flex-wrap: nowrap;
+
+			& > *:not(:first-child) {
+				margin-inline-start: var(--ck-spacing-standard);
+			}
+		}
```

Comment:
Move those properties to the `ckeditor5-ui/theme/components/formrow/formrow.css`.

Reply:
Addressed in c488617.